src/Bundle/Resources/skeleton/Factory.tpl.php

```diff
@@ -10,6 +10,7 @@
 use Zenstruck\Foundry\Proxy;
 
 /**
+ * @method static <?= $entity->getShortName() ?>|Proxy createSingle(array $attributes = [])
```

Comment:
`createMany` below should be marked as `static` now, right?

Reply:
Or will that typehint interfere with the non-static createMany() `@method` on `Factory`?

Reply:
I've now made it static, didn't try autocompletion yet (I don't normally use an IDE when working on small projects like this)

Reply:
Regarding auto-completion, I think the'll be some issues mixing static and non-static `@method`'s of the same name. At least with PhpStorm.

Reply:
As the instance method is deprecated, should we only document the static method in the PHPdoc?

Reply:
I've updated the PR with my latest suggestion

Reply:
Not ideal but I think this is best.